### ISSUE TITLE
Not working on Catalina 10.15.3

### ISSUE THREAD

**Issue by UninspiredCoder** [DaFuqtor/MiddleClick-Catalina#20]
Already tried downloading the program and also with brew, but it just doesn't work. Tried kickall, restarting, adding/removing program from privacy/accessibility. 
Any idea?

Thanks

**Comment by DaFuqtor** [comment]
Works on my MacBook Air 2018 with macOS 10.15.3.

Could you tell us a little more about the problem? The program doesn't start, or maybe it starts but doesn't perform its function?

**Comment by isiachi** [comment]
After I have updated with `brew` from `2.4.5` to `2.4.7`, I run into the same issue. I have solved by unchecking and checking the `accessibility` permission. It seems the signing key of 2.4.5 is different from the one used in 2.4.7 (if I remember right).

**Comment by UninspiredCoder** [comment]
> Works on my MacBook Air 2018 with macOS 10.15.3.
> 
> Could you tell us a little more about the problem? The program doesn't start, or maybe it starts but doesn't perform its function?

It starts but doesn’t do anything. Permissions are ok in privacy. Tried with Magic Mouse (1st gen, with AA batteries and also with the trackpad). MacBook Pro mid 2012 non retina, 8gb ram.
Maybe I could check log files if you point me in the right direction

**Comment by UninspiredCoder** [comment]
> After I have updated with `brew` from `2.4.5` to `2.4.7`, I run into the same issue. I have solved by unchecking and checking the `accessibility` permission. It seems the signing key of 2.4.5 is different from the one used in 2.4.7 (if I remember right).

Already tried that but didn’t help.

**Comment by DaFuqtor** [comment]
> > After I have updated with `brew` from `2.4.5` to `2.4.7`, I run into the same issue. I have solved by unchecking and checking the `accessibility` permission. It seems the signing key of 2.4.5 is different from the one used in 2.4.7 (if I remember right).
> 
> Already tried that but didn’t help.

Try completely deleting and readding it to `Accessibility` preferences. That's what I've done for it to work after update.

**Comment by CartoonChess** [comment]
I had this problem as well updating from 2.4.5 to 2.4.7 downloading via the releases page. I tried toggling the accessibility preference as well as deleting the app altogether, but this didn't solve the problem. It appeared to be repeatedly opening and closing, and Activity Monitor showed there were often two processes running with the same name, even after force quitting both. I had to keep force quitting them over and over for about a minute until it was finally "happy." Seems to be fine now.

Also of note, Catalina still complained that it couldn't verify the app as being malicious software or not.

**Comment by patpilus** [comment]
I can confirm that re-checking the Accessibility permission solves the problem.

**Comment by guillermotti** [comment]
I can confirm it also, it's working in Catalina 10.15.3

**Comment by popa-marius** [comment]
One more confirm that re-adding the Accessibility permission solves the problem.

**Comment by UninspiredCoder** [comment]
Well for me it still doesn't work. I've tried that many times with no results.

**Comment by DaFuqtor** [comment]
> Well for me it still doesn't work. I've tried that many times with no results.

Hmm. How exactly are you trying to get it to work? I mean, what's your "palm action"?

**Comment by UninspiredCoder** [comment]
First I tried installing via direct download, after that providing the required permissions on the system preferences. It didn't work, so after seeing some messages in this forum, I tried removing and adding the permission in Privacy - Accessibility. It didn't work, so I removed the program and re-installed it with brew. I did the same than with the direct download but no changes. Now after seeing the messages in this thread tried that again, but no results. I also tried removing and adding the middleclick from the accessibility list (instead of just unchecking and checking). 

I'm using the first version of magic mouse...could that be a problem?

**Comment by CartoonChess** [comment]
I also had a hell of a time getting the new version to work, as I detailed above. I’m also using the original Magic Mouse.
